.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @manudeli @okinawaa @minsoo-web
```

Comment:
What is CODEOWNER’s DRI? In my opinion, It is to strictly review all PRs to be merged into main and ensure that the library is distributed reliably to library users and that the development process flows smoothly.

Recently, @okinawaa and @minsoo-web have had their own DRIs, so I will remove them from CODEOWNERS, but I will remember you on the contributors page. In this repository, but we maintain maintain privileges as collaborators so that you can help with Suspensive yet.

When the company inquired with the Head of frontend, they responded that external maintainers (@gwansikk, @kangju2000) do not have CODEOWNERS privileges. This means that merging into main is only possible through @manudeli.